src/test/java/org/fundacionjala/coding/maria/Kata8Test.java

```diff
@@ -0,0 +1,98 @@
+package org.fundacionjala.coding.maria;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Kata 8.
+ */
+public class Kata8Test {
+    private Kata8 kata8;
+    private static final String COMMONLINE2 = "    _  _     _  _  _  _  _ ";
+    private static final String COMMONLINE3 = "  ||_  _|  | _||_|  ||_| _|";
+    private static final String COMMONSTRING = "123456789";
+
+    /**
+     * initialize kata8.
+     */
+    @Before
+    public void initialize() {
+        kata8 = new Kata8();
+    }
+
+    /**
+     * testBankOCRNormal test 1.
+     */
+    @Test
+    public void testBankOCRNormal() {
+        String filaDos =  "  | _| _||_||_ |_   ||_||_|";
+        assertEquals(COMMONSTRING, kata8.bankOCR(COMMONLINE2, filaDos, COMMONLINE3));
+    }
+
+    /**
+     * testBankOCRNormalInt test 1.
+     */
+    @Test
+    public void testBankOCRNormalInt() {
+        String filaDos =  "  | _| _||_||_ |_   ||_| _ ";
+        assertEquals("12345678?", kata8.bankOCR(COMMONLINE2, filaDos, COMMONLINE3));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "12345678?" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)